src/Catalyst.Node.POA.CE/Program.cs

```diff
@@ -98,6 +102,21 @@ private static async Task CustomBootLogic(Kernel kernel)
             RegisterNodeDependencies(Kernel.ContainerBuilder);
 
             kernel.StartContainer();
+
+            //var registrations = kernel.Instance.ComponentRegistry.Registrations;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)